src/main/java/com/alkemy/ong/security/JwtUtils.java

```diff
@@ -0,0 +1,123 @@
+package com.alkemy.ong.security;
+
+import io.jsonwebtoken.Claims;
+import io.jsonwebtoken.Jwts;
+import io.jsonwebtoken.SignatureAlgorithm;
+
+import java.util.Date;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.function.Function;
+
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.security.core.userdetails.UserDetails;
+import org.springframework.stereotype.Service;
+
+@Service
```

Comment:
Please make this class a Component. We mark beans with @Service to indicate that they're holding any business logic. See: https://www.baeldung.com/spring-component-repository-service